_common

```diff
@@ -31,13 +31,19 @@ job_ids() {
 # an error, and the calling command and line
 runcli() {
     local args=("$@")
-    local rc output
+    local rc output errorlog errstr=
     set +e
-    output=$("${args[@]}" 2>&1)
+    errorlog=$(mktemp -t runcli-errorlog-XXXX)
```

Comment:
if you use a temporary file you should still ensure it's deletion, e.g. with

```
trap "rm $errorlog" EXIT
```

Reply:
done.
I just realized that running the test suite (`test/02-investigate.t`) leaves several tempfiles around, not named specifically (e.g. `/tmp/tmp.1BrG2J6Atc`). Not critical but I would like to know why they aren't deleted

Reply:
Looks like it's from 
```
# Make auto-deleting temp files for testing subprocesses:
+fs:mktemp; comment_1234_updated=$temp
...
```
so I'l poke @ingydotnet about it

Reply:
Looking into it.

Reply:
I think I know the problem. 
The bashplus function `+fs:mktemp` sets a trap to auto-remove it. 
But in bash you can only have one trap per signal per process.
Bashplus also has a `+trap` function that allows you to have multiple traps.
But if you use a regular `trap` after `+trap` has been used it will blow it away.
Can you try changing the `trap` you just added to `+trap`?
I think that would solve the issue.

Bashplus could probably wrap the `trap` function to warn about this, but I'd need to think more about it.